src/WebApp/Extensions/Extensions.cs

```diff
@@ -98,14 +98,13 @@ public static void AddAuthenticationServices(this IHostApplicationBuilder builde
 
     private static void AddAIServices(this IHostApplicationBuilder builder)
     {
-        string? ollamaEndpoint = builder.Configuration["AI:Ollama:Endpoint"];
-        if (!string.IsNullOrWhiteSpace(ollamaEndpoint))
+        if (builder.Configuration["OllamaEnabled"] is string ollamaEnabled && bool.Parse(ollamaEnabled))
         {
-            builder.Services.AddChatClient(new OllamaChatClient(ollamaEndpoint, builder.Configuration["AI:Ollama:ChatModel"] ?? "llama3.1"))
+            builder.AddKeyedOllamaSharpChatClient("chat");
```

Comment:
Why `keyed`?

Reply:
Hold over from a bug that I had where registrations weren't happening correctly in the Toolkit. That bug is fixed in 9.1 of the Toolkit so I've dropped the keyed service here (and in the catalog API)